src/lib/Commands/TestCommand/index.js

```diff
@@ -315,17 +363,20 @@ imp.wakeup(${this.startupDelay /* prevent log sessions mixing, allow service mes
     // quote file name for line control statement
     const quoteFilename = f => f.replace('"', '\\"');
 
+    let tmpFrameworkFile = this.testFrameworkFile.replace(new RegExp('\\\\', 'g'),"/");
```

Comment:
1) The code is tested on win7
2) I can't say why, but the /\\\\/g doesn't work on win7:
Including source "C:sersmitryppDataoaming
pm
ode_modulesmptestrcli/../impUnit/index.nut"